eo-maven-plugin/src/main/java/org/eolang/maven/Home.java

```diff
@@ -0,0 +1,246 @@
+        return this.write(new InputOf(str));
+    }
+
+    /**
+     * Saving text.
+     * @param txt Text
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final Text txt) throws IOException {
+        return this.write(new InputOf(txt));
+    }
+
+    /**
+     * Saving stream.
+     * @param stream Input stream
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final InputStream stream) throws IOException {
+        return this.write(new InputOf(stream));
+    }
+
+    /**
+     * Saving input.
+     * @param input Input
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final Input input) throws IOException {
+        return this.write(input);
+    }
+
+    /**
+     * Saving bytes.
+     * @param bytes Byte array
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final byte[] bytes) throws IOException {
+        return this.write(new InputOf(bytes));
+    }
+
+    /**
+     * Saving string.
+     * @param str String
+     * @return Saved bytes
+     */
+    public long saveQuietly(final String str) {
+        try {
+            return this.write(new InputOf(str));
+        } catch (final IOException ex) {
+            throw new IllegalStateException(ex);
+        }
+    }
+
+    /**
+     * Saving text.
+     * @param txt Text
+     * @return Saved bytes
+     */
+    public long saveQuietly(final Text txt) {
+        try {
+            return this.write(new InputOf(txt));
+        } catch (final IOException ex) {
+            throw new IllegalStateException(ex);
+        }
+    }
+
+    /**
+     * Saving input stream.
+     * @param stream InputStream
+     * @return Saved bytes
+     */
+    public long saveQuietly(final InputStream stream) {
+        try {
+            return this.write(new InputOf(stream));
+        } catch (final IOException ex) {
+            throw new IllegalStateException(ex);
+        }
+    }
+
+    /**
+     * Saving bytes.
+     * @param bytes Byte array
+     * @return Saved bytes
+     */
+    public long saveQuietly(final byte[] bytes) {
+        try {
+            return this.write(new InputOf(bytes));
+        } catch (final IOException ex) {
+            throw new IllegalStateException(ex);
+        }
+    }
+
+    /**
+     * Saving input.
+     * @param input Input
+     * @return Saved bytes
+     */
+    public long saveQuietly(final Input input) {
+        try {
+            return this.write(input);
+        } catch (final IOException ex) {
+            throw new IllegalStateException(ex);
+        }
+    }
+
+    /**
+     * Write to file.
+     * @param stream Input
+     * @return Written bytes
+     * @throws IOException if fails
+     */
+    public long write(final Input stream) throws IOException {
+        final File dir = this.cwd.toFile().getParentFile();
```

Comment:
@yasamprom `cwd` means "current working directory". Why it's a file to write? I believe, `Home` has wrong design. Should be:

```
Home {
  Path cwd;
  void save(Path, String);
  String load(Path);
  String rel(Path);
}
```

Reply:
@yegor256 let's name it `path` instead of cwd? I don't see reasons to add second argument in `save` because we  don't have use cases where we need to have `cwd` and `file` separately.

Reply:
> @yegor256 let's name it `path` instead of `cwd`? I don't see reasons to add second argument in `save` because we don't have use cases where we need to have `cwd` and `file` separately.

And if we need it some day we will always be able to use `new Home(<needed path>)`

Reply:
> > @yegor256 let's name it `path` instead of `cwd`? I don't see reasons to add second argument in `save` because we don't have use cases where we need to have `cwd` and `file` separately.
> 
> We can always use `new Home(<needed path>)`

Well, then the `Home` name will be strange. I will add `path` argument to`save`

Reply:
@yasamprom `cwd` will help us print log messages nicely, because we will know where is the "home" and how relevant is the `path` to it